airsonic-main/src/main/java/org/airsonic/player/service/AudioScrobblerService.java

```diff
@@ -97,7 +99,7 @@ public synchronized void register(MediaFile mediaFile, String username, boolean
         try {
             queue.put(registrationData);
         } catch (InterruptedException x) {
-            LOG.warn("Interrupted while queuing Last.fm scrobble.", x);
+            LOG.warn("Interrupted while queuing Last.fm scrobble: " + x.toString());
```

Comment:
Is that still uniquely used by Last.fm? Shouldn't that text be changed to remove that?